src/uikit/Card.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import styled from 'react-emotion';
+import { Flex } from './Core';
+
+const CardContainer = styled(Flex)`
+  border-radius: 10px;
+  background-color: #ffffff;
+  border: solid 1px #e0e1e6;
+  flex: 1;
+  min-height: 440px;
+`;
+
+const Card = ({ children, ...props }) => <CardContainer {...props}>{children}</CardContainer>;
```

Comment:
what is this wrapper for? doesn't seem like it's doing much

Reply:
Extracted it to be used for the other widgets which all common 'box'